httpd/plan.sh

```diff
@@ -12,18 +12,21 @@ pkg_build_deps=(core/patch core/make core/gcc)
 pkg_bin_dirs=(bin)
 pkg_lib_dirs=(lib)
 pkg_expose=(80 443)
-pkg_svc_run="bin/httpd -DFOREGROUND -f $pkg_svc_config_path/httpd.conf"
 pkg_svc_user="root"
+pkg_svc_group="root"
 
 do_build() {
-  ./configure --prefix="$pkg_prefix" \
-              --with-expat="$(pkg_path_for expat)" \
-              --with-iconv="$(pkg_path_for libiconv)" \
-              --with-pcre="$(pkg_path_for pcre)" \
-              --with-apr="$(pkg_path_for apr)" \
-              --with-apr-util="$(pkg_path_for apr-util)" \
-              --with-z="$(pkg_path_for zlib)" \
-              --enable-ssl --with-ssl="$(pkg_path_for openssl)" \
-              --enable-modules=most --enable-mods-shared=most
-  make
+    ./configure --prefix="$pkg_prefix" \
+                --with-expat="$(pkg_path_for expat)" \
+                --with-iconv="$(pkg_path_for libiconv)" \
+                --with-pcre="$(pkg_path_for pcre)" \
+                --with-apr="$(pkg_path_for apr)" \
+                --with-apr-util="$(pkg_path_for apr-util)" \
+                --with-z="$(pkg_path_for zlib)" \
+                --with-ssl="$(pkg_path_for openssl)" \
+                --enable-modules="none" \
+                --enable-mods-static="unixd" \
+                --enable-mods-shared="reallyall" \
```

Comment:
Thanks for the review, here are my thoughts:

To the question: "Why the change to a run hook instead of using this?"
Creating a run hook file rather than letting habitat automatically build one allows changes to the run file post-build rather than having to rebuild httpd.

To the question: "What benefit do we get from compiling this static instead of using a module?"
enable-mods-static="unixd" adds one static module at build time called unixd which is necessary for runtime changes to httpd user. It is a necessary module for httpd to run and by building it statically it will always be linked to the binaries rather than relying on the user to link it dynamically.

To the question: "Why the change here, and --enable-modules="none" above?"
enable-modules="none" does not add any modules dynamically at build time so that only the modules the user chooses in their configuration file will be added at runtime.

In addition, thank you for your comments and concerns and I will fix the problems you brought up including the formatting problem.